### ISSUE TITLE
Add option to Autoupdate plugin

### ISSUE THREAD

**Issue by michaeltorbert** [semperfiwebdesign/all-in-one-seo-pack#1524]
Look into ones like WordFence, etc

**Comment by EkoJr** [comment]
I've actually discussed a similar subject on WP's Slack about auto-updating plugins, and allowing plugin authors to ease into the feature; instead of forcefully auto-updating, but at the same time, letting plugin users know there is an update available (either as an RC, Beta, or Hotfix).

This topic mostly introduced a beta concept. [Beta Download/Install capabilities with Plugins & Themes](https://core.trac.wordpress.org/ticket/42376).

Another concept I have drafted, but haven't posted the issue onto WordPress Trac yet. Is the ability to allow plugin/theme authors to force updates from target version bugs & security issues.

So, essentially for AIOSEOP, it may be possible to introduce similar concepts by checking the semantic version. This would probably help prevent high volumes of issues from both old and new versions (once the concept is introduced).
